packages/eslint-plugin/src/rules/member-ordering.ts

```diff
@@ -947,20 +991,24 @@ export default createRule<Options, MessageIds>({
           );
 
           if (grouped == null) {
+            runSort(members);
             return false;
           }
 
           if (hasAlphaSort) {
-            return !grouped.some(
-              groupMember =>
-                !checkAlphaSort(groupMember, order as AlphabeticalOrder),
-            );
+            const sortResults: boolean[] = [];
+            grouped.forEach(groupMember => {
+              sortResults.push(
+                checkAlphaSort(groupMember, order as AlphabeticalOrder),
+              );
+            });
+            return sortResults.some(result => !result);
```

Comment:
[Style] No need to create a separate array and then `.forEach`; `.map` works the same.

```suggestion
            return grouped
              .map(groupMember =>
                checkAlphaSort(groupMember, order as AlphabeticalOrder),
              )
              .some(result => !result);
```